slither/detectors/operations/missing_initialization.py

```diff
@@ -0,0 +1,103 @@
+"""
+Module detecting missing initialization used to check conditions.
+
+"""
+
+from slither.detectors.abstract_detector import AbstractDetector, DetectorClassification
+
+
+class MissingInitialization(AbstractDetector):
+    """
+    Detect missing-initialization
+    """
+
+    ARGUMENT = "missing-initialization"  # slither will launch the detector with slither.py --mydetector
+    HELP = "Initialize function misses initialization"
+    IMPACT = DetectorClassification.HIGH
+    CONFIDENCE = DetectorClassification.HIGH
+
+    WIKI = "https://github.com/trailofbits/slither/wiki/Detector-Documentation#missing-initialization"
+    WIKI_TITLE = "MISSING_INITIALIZATION"
+    WIKI_DESCRIPTION = "MISSING_INITIALIZATION"
+    WIKI_EXPLOIT_SCENARIO = """
+```solidity
+contract A {
+    uint public state_variable = 0;
+    bool public initialized = false;
+
+    modifier not_initialized(){
+        require(initialized == false);
+        _;
+    }
+
+    function initialize(uint _state_variable) public not_initialized {
+        state_variable = _state_variable;
+    }
+}
+```
+Bob calls `initialize`. However, Alice can also call `initialize`.
+"""
+    WIKI_RECOMMENDATION = (
+        "Check state variable should be initialized in initialize function"
+    )
+
+    def detect_initilize(self, func):
```

Comment:
`detect_initilize` sould probably be `detect_initialize`

Reply:
Thanks